config/main.py

```diff
@@ -2199,12 +2199,15 @@ def generate_sysinfo(cur_config, config_input, ns=None):
 
     mac = None
     platform = None
+    asic_id = None
     cur_device_metadata = cur_config.get('DEVICE_METADATA')
 
     # Reuse current config's mac and platform. Generate if absent
```

Comment:
>platform [](http://example.com/codeflow?start=37&length=8)

minor comment: this code comment should be updated as code, or make it more generic.